synapse/storage/databases/main/event_push_actions.py

```diff
@@ -1255,7 +1255,6 @@ def __init__(
             table="event_push_actions",
             columns=["highlight", "stream_ordering"],
             where_clause="highlight=0",
-            psql_only=True,
```

Comment:
#13141 gives no info on why this is PostgreSQL-only, but I assume because the partial index was ignored on SQLite?